src/Ancestor/Ancestor.DataAccess/3.DAO/LambdaExpressionHelper.cs

```diff
@@ -409,6 +409,15 @@ protected override Expression VisitMethodCall(MethodCallExpression m)
                         this.Visit(m.Object);
                         this.Write(")");
                         return m;
+                    case "ToString":
+                        this.Write("TO_CHAR(");
```

Comment:
Should check dbtype while using TO_CHAR
Becuase Oracle use TO_CHAR, but MSSQL use CHAR